hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/workflows/PreHandleContext.java

```diff
@@ -284,4 +285,14 @@ PreHandleContext requireKeyIfReceiverSigRequired(
     @NonNull
     TransactionKeys allKeysForTransaction(@NonNull TransactionBody body, @NonNull AccountID payerId)
             throws PreCheckException;
+
+    @NonNull
+    void executeInnerPreHandle(@NonNull TransactionBody body, @NonNull AccountID payerId) throws PreCheckException;
+
+    /**
+     * Returns the TransactionBogy from the given transaction.
+     * @return the TransactionBogy
+     */
+    @Nullable
+    TransactionBody bodyFromTransaction(@NonNull final Transaction tx) throws PreCheckException;
```

Comment:
These both are no more needed. All the inner transactions go through pre-handle with my PR changes that are merged